src/eei.h

```diff
@@ -116,8 +116,29 @@ struct EthereumInterface : ShellExternalInterface {
 };
 
 struct GasSchedule {
+  static constexpr unsigned storageLoad = 200;
   static constexpr unsigned storageStoreCreate = 20000;
   static constexpr unsigned storageStoreChange = 5000;
+  static constexpr unsigned log = 375;
+  static constexpr unsigned logData = 8;
+  static constexpr unsigned logTopic = 375;
+  static constexpr unsigned create = 32000;
+  static constexpr unsigned call = 700;
+  static constexpr unsigned copy = 3;
+  static constexpr unsigned blockhash = 800;
+  static constexpr unsigned balance = 400;
+  static constexpr unsigned stepGas0 = 0;
+  static constexpr unsigned stepGas1 = 2;
+  static constexpr unsigned stepGas2 = 3;
+  static constexpr unsigned stepGas3 = 5;
+  static constexpr unsigned stepGas4 = 8;
+  static constexpr unsigned stepGas5 = 10;
+  static constexpr unsigned stepGas6 = 20;
+  static constexpr unsigned stepGas7 = 0;
```

Comment:
Why 0?

Reply:
This was here:
https://github.com/ethereum/evmjit/blob/develop/libevmjit/JIT.h#L71

It is unneeded, so I am getting rid of it